SIS_import/sis_upload.rb

```diff
@@ -90,6 +114,9 @@ def Canvas_API_POST(url, post_params)
 
 ## make Canvas API POST call
 def Canvas_API_PUT(url, post_params)
+	# make sure the Canvas call is within API usage quota
```

Comment:
Aren't these put_params?  It make confuse people who read the code if they have incorrect names.  Maybe they should just be params